specification/frontdoor/resource-manager/readme.md

```diff
@@ -31,6 +31,24 @@ openapi-type: arm
 tag: package-2020-05
 ```
 
+### Tag: package-2020-07
+
+These settings apply only when `--tag=package-2020-07` is specified on the command line.
+
+``` yaml $(tag) == 'package-2020-07'
+input-file:
+- Microsoft.Network/stable/2020-07-01/network.json
+- Microsoft.Network/stable/2020-07-01/networkexperiment.json
+- Microsoft.Network/stable/2020-07-01/frontdoor.json
+- Microsoft.Network/stable/2020-07-01/webapplicationfirewall.json
```

Comment:
In 05-01 we had issues generating the SDK when we upgraded the WAF version. The WAF 05-01 version wasn't added to the manifest and the checks/tests failed, so we [reverted back to the 04 version](https://github.com/Azure/azure-rest-api-specs/pull/9964). If WAF hasn't changed since 04 then let's keep it, otherwise if the manifest has been updated we can bump up the version

Reply:
The error schema has been changed in 07-01